### ISSUE TITLE
[TST] No unit tests, no CI

### ISSUE THREAD

**Issue by matteobachetti** [xragua/xraybinaryorbit#3]
The code has no unit testing and continuous integration for code tests, which is something required by JOSS.
Guidelines for writing tests I found very useful when I started doing this, are, again, from Astropy (many hearts to Astropy): https://docs.astropy.org/en/latest/development/testguide.html#writing-tests

**Comment by xragua** [comment]
Thank you very much for the link! I'm currently working on implementing unit tests and continuous integration, and I really appreciate the guidelines from Astropy :). I will let you know once I’ve made progress on getting everything up and running.

**Comment by xragua** [comment]
Dear @matteobachetti, I have implemented the CI test. To facilitate this, I made some adjustments to the code. Now, functions interacting with the tinker form can load directly from the .txt file in the directory by setting load_directly=True. If the .txt file does not exist and load_directly=True, the form will still be displayed. Alternatively, if the .txt exists and load_directly=False, it will load and be available for modifications if needed. Thank you!
